### ISSUE TITLE
[Question] [Feature Request] GUI change and select presenter as me.

### ISSUE THREAD

**Issue by Maniues** [Anapher/Strive#8]
# Hello!

### Firstly, thanks to the author! You are doing amazing thing. My school uses bbb, I don't like it, because I hate Java and bbb is hard to install on server. This software is simply and it's the best OSS alternative for bbb I found.

But, I think this GUI is a bit uncomfortable. 
My questions:

- Where I can found a HTML/CSS/JS files to change layout?
- Is available to make me as a presenter?

Feature request:
- If answer to the second question is "no", please add that option.
- Add an option to select layout of the elements.

Regardless of the above, this software is amazing and is the only viable OSS conferencing software written in C #

**Comment by Anapher** [comment]
Hello,
thank you very much for your comments.

> But, I think this GUI is a bit uncomfortable.

Do you mind elaborating on this? I'm always striving to improve Strive.

> Where I can found a HTML/CSS/JS files to change layout?

I'm using the library [Material-UI](https://mui.com/), which includes a CSS-In-JSX solution: https://mui.com/styles/basics/
You can find the frontend code here: https://github.com/Anapher/Strive/tree/develop/src/Web/WebSPA

> Is available to make me as a presenter?

Actually no, but if you are a moderator, you automatically have the benefits of being a presenter, meaning you can screen share etc. by default

> Add an option to select layout of the elements.

Can you eloborate on this? Which elements do you mean?

**Comment by Maniues** [comment]
> Do you mind elaborating on this? I'm always striving to improve Strive.

And

> Can you eloborate on this? Which elements do you mean?

Okay, I will be honest. I prefer the layout from bbb. It's clear, and simple. I think it's just a matter of habit, and your layout is is technically better. I think, it's only the habit. Some elements are more complicated for me now. e.g Permissions. How does it work. There are two options: standard checkbox (on the left) and the second checkbar on the right. It's not clear for user.
To make this software more popular or even usable for tutors, you should provide a "migration options". So, Strive should implement interface similar to MS Teams and bbb. Not a copy, but similar.

My concept:
You can add many (or at least 2) layouts e.g bbb style for tutors that migrate from bbb, ms teams layout and the default layout. In account options, you can choose your layout.
The default layout should be the current layout).

> Actually no, but if you are a moderator, you automatically have the benefits of being a presenter, meaning you can screen share etc. by default.

But you are no in the center of presentation. In live demo is not a problem, because you can log to another account and set it to presenter, but for private instances it may be problem.

**Comment by Anapher** [comment]
> Some elements are more complicated for me now. e.g Permissions

I agree with you that permissions is currently quite a complicated feature, but I think you actually don't really need to use it that much as most of the stuff is handled automatically (e.g. a moderator has by default all permissions and a presenter gets the permissions while he is in that role).

But I will absolutely work on a user documentation 👍 

> So, Strive should implement interface similar to MS Teams and bbb. Not a copy, but similar.

The problem is that supporting and maintaining several user interfaces is quite complicated and time consuming to keep everything up to date. Also, it makes documentation even more complicated as I would have to include screenshots from every layout to provide support for everyone. Currently, I understand the layout as a mix from Zoom, BBB and Teams. 

> But you are no in the center of presentation

You are right, I will add this feature.

**Comment by Maniues** [comment]
> But I will absolutely work on a user documentation 👍

Oh, it's fantastic! 

By the way, I recommend that the documentation be dual-licensed under CC-BY 4.0 (or 3.0) and Apache instead of only Apache. This will facilitate the use of this documentation.

> The problem is that supporting and maintaining several user interfaces is quite complicated and time consuming to keep everything up to date. Also, it makes documentation even more complicated as I would have to include screenshots from every layout to provide support for everyone. Currently, I understand the layout as a mix from Zoom, BBB and Teams.

Oh, yes. I understand. I must be hard to implement. Because of this, I suggest making video on youtube and place it here on github as a file, that describes Strive inteface (and I hope it will be cc-by licensed too. cc-by can be chose as a licensing option on youtube - sorry for talking about copyrights).
